tools/compare_pages.sh

```diff
@@ -16,6 +16,7 @@ help() {
     Optional Arguments:
       -p, --printable               Compares your build against 'printable' build.
       -s, --single-page             Combines all compared pages into a single image.
+      -m, --mapping A:B             Page A on the left will be with page B on the right.
```

Comment:
Please add an example, as to me it's not evident how to use it. What if I want to create 5 images, and all of them offset by 1 page?

Reply:
Done